vars/main.yml

```
@@ -13,9 +13,10 @@ core:
     access_subnet: "192.168.252.1/24"	# access subnet & gateway
     core_subnet: "192.168.250.1/24"	# core subnet & gateway
     mode: af_packet			# Options: af_packet or dpdk
+    multihop_gnb: false                 # when set to true, access subnet will be same as data_iface
```

@sureshmarikkannu, I think the comment should be the opposite. That is, 
```suggestion
    multihop_gnb: false                 # when set to true, external gNB may be on the same subnet as data_iface or multiple hops away
```